scripts/delayed.js

```diff
@@ -1,7 +1,13 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { sampleRUM, loadScript } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+loadScript('/scripts/video.js', () => {
```

Comment:
We should load the video.js from the original host here, no need to copy this

Reply:
Makes sense